server/src/db/index.ts

```diff
@@ -125,6 +155,7 @@ export async function getEvent(id: number | string): Promise<null | PoapEvent> {
       ...res,
       start_date: formatDate(res.start_date),
       end_date: formatDate(res.end_date),
+      event_template_id: res.event_template_id,
```

Comment:
If the `publicEventColumns` includes `event_template_id`, why are you including it?
Remember that `...res` means that is copying all elements from the object, and the overwriting (or creating) `start_date` and `end_date`